src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -494,6 +495,32 @@ class spell_mage_glyph_of_eternal_water : public SpellScriptLoader
     }
 };
 
+class spell_mage_combustion_proc : public SpellScriptLoader
+{
+public:
+    spell_mage_combustion_proc() : SpellScriptLoader("spell_mage_combustion_proc") {}
+
+    class spell_mage_combustion_proc_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_mage_combustion_proc_AuraScript);
+
```

Comment:
```suggestion

    bool Validate(SpellInfo const* /*spellInfo*/) override
    {
        return ValidateSpellInfo({ SPELL_MAGE_COMBUSTION });
    }

```
You should always validate spells used within a script